src/getCoinCombination.test.js

```diff
@@ -7,4 +7,64 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it(`should return 0 penny for 0 cent`, () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+  });
+
+  it(`should return 1 penny for 1 cent`, () => {
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
+  });
+
+  it(`should return 3 pennies for 3 cents`, () => {
+    expect(getCoinCombination(3)).toEqual([3, 0, 0, 0]);
+  });
+
+  it(`should return 1 nickel for 5 cents`, () => {
+    expect(getCoinCombination(5)).toEqual([0, 1, 0, 0]);
+  });
+
+  it(`should return 1 nickel and 2 pennies for 7 cent`, () => {
+    expect(getCoinCombination(7)).toEqual([2, 1, 0, 0]);
+  });
+
+  it(`should return 1 dime for 10 cent`, () => {
+    expect(getCoinCombination(10)).toEqual([0, 0, 1, 0]);
+  });
+
+  it(`should return 1 dime and 3 pennies for 13 cents`, () => {
+    expect(getCoinCombination(13)).toEqual([3, 0, 1, 0]);
+  });
+
+  it(`should return 1 dime and 1 nickels for 15 cents`, () => {
```

Comment:
There is a typo in the test description: '1 dime and 1 nickels' should be '1 dime and 1 nickel' to match the expected output.